### PR TITLE
Provide correct reference link.

### LINKED ISSUE #237: Ep. 09: Broken link to date/time values discussion
The link referring to a discussion on date/time values in [Ep. 09 Creating and Modifying Data](http://swcarpentry.github.io/sql-novice-survey/09-create/) doesn't work (links back to the same lesson).

> Most databases also support Booleans and date/time values; SQLite uses the integers 0 and 1 for the former, and represents the latter as discussed [earlier](http://swcarpentry.github.io/sql-novice-survey/09-create/#a:dates). 

As far as I can tell, date/time is discussed in:
1. the **Date Types** callout in [Ep. 03 Filtering](http://swcarpentry.github.io/sql-novice-survey/03-filter/)
2. the solution for the **Identifying Atomic Values** problem in [Ep. 08 Data Hygiene](http://swcarpentry.github.io/sql-novice-survey/08-hygiene/).

Can this be consolidated?

### PR DESCRIPTION
**Pull request by henrykironde**
fixes #237

### CODE REVIEW

File: _episodes/09-create.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ but most provide the following:
 
 Most databases also support Booleans and date/time values;
 SQLite uses the integers 0 and 1 for the former,
-and represents the latter as discussed [earlier](#a:dates).
+and represents the latter as discussed [earlier]({{ site.github.url }}/03-filter/).
```
Comment (remram44) [inline _episodes/09-create.md]: Should we link to the anchor `#date-types`?

Comment (henrykironde) [reply _episodes/09-create.md]: Where does `#date-types` resolve to. I can't seem to find anything related to #date-types

Comment (remram44) [reply _episodes/09-create.md]: https://swcarpentry.github.io/sql-novice-survey/03-filter/index.html#date-types

**Comment by henrykironde** [comment]
I am not really sure where that is coming from. f30cc4e. Any idea?

**Pull request merged**